pkgs/applications/version-management/git-and-tools/git-quick-stats/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, fetchFromGitHub, makeWrapper, coreutils, git, gnugrep, gnused }:
+
+stdenv.mkDerivation rec {
+    name = "git-quick-stats-${version}";
+    version = "2.0.8";
+
+    src = fetchFromGitHub {
+        owner = "arzzen";
+        repo = "git-quick-stats";
+        rev = "c11bce17bdb1c7f1272c556605b48770646bf807";
+        sha256 = "1px1sk7b6mjnbclsr1jn33m9k4wd8wqyw4d6w1rgj0ii29lhzmqi";
+    };
+
+    buildInputs = [ makeWrapper ];
+
+    dontBuild = true;
+
+    installPhase = ''
+        mkdir -p $out/bin
+        cp -a git-quick-stats $out/bin/git-quick-stats
+    '';
+
+    wrapperPath = with stdenv.lib; makeBinPath [
+        coreutils
+        git
+        gnugrep
+        gnused
+    ];
+
+    fixupPhase = ''
+        patchShebangs $out/bin
+
+        wrapProgram $out/bin/git-quick-stats --prefix PATH : "${wrapperPath}"
+    '';
+
+    meta = {
```

Comment:
Also missing a maintainers attribute

Reply:
Should I add myself as a package maintainer (in derivation AND maintainers-list.nix)?

Reply:
Sorry missed this comment, yes you should